buildtasks/sync.js

```diff
@@ -0,0 +1,32 @@
+//******************************************************************************
+//* sync.js
+//*
+//* Defines a custom gulp task for serving up content from the server-root 
+//* local folder, setup file/folder watchers so that changes are reflected
+//* on file save, and open the default browser to the default html page. 
+//******************************************************************************
+
+"use strict";
+
+//******************************************************************************
+//* DEPENDENCIES
+//******************************************************************************
+
+var gulp = require("gulp");
+ 
+
+function copyJstosharepoint(buildServeTaskName) {
+   
+
+    gulp.watch(global.TSWorkspace.Files, ["lint", buildServeTaskName]);
+//    gulp.watch(global.PnPLocalServer.RootFolder).on('change', browserSync.reload);
```

Comment:
no, that was on purpose. Task is just meant to copy to sharepoint.

On Sun, May 1, 2016 at 7:39 AM, Patrik Björklund <notifications@github.com>
wrote:

> In buildtasks/sync.js
> <https://github.com/OfficeDev/PnP-JS-Core/pull/79#discussion_r61685911>:
>
> > +//******************************************************************************
> > +
> > +"use strict";
> > +
> > +//******************************************************************************
> > +//* DEPENDENCIES
> > +//******************************************************************************
> > +
> > +var gulp = require("gulp");
> > +
> > +
> > +function copyJstosharepoint(buildServeTaskName) {
> > +
> > +
> > +    gulp.watch(global.TSWorkspace.Files, ["lint", buildServeTaskName]);
> > +//    gulp.watch(global.PnPLocalServer.RootFolder).on('change', browserSync.reload);
>
> Accidentally commented out?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OfficeDev/PnP-JS-Core/pull/79/files/3cbaa261d07af86524caebb868cbc803b373beda#r61685911>
>